hicommonwealth/commonwealth
# Add possibility to reserve namespace without enabling stake

## Link to Issue
Closes: #9513
Closes: #9515
Closes: #9551 
Closes: #9552

## Description of Changes
- modified existing flow so that reserve namespace step can be done by itself without needing of enabling stake
- additionally fixed some small copy/logic issues related to weighted voting

## Test Plan
- FLAG_WEIGHTED_TOPICS=true
- go to create community flow
- if you select chain from the dropdown that supports stake, you should see `namespace` step as a third one
- you can skip namespace step in create community flow 
- go to topics and try to create a new weighted topic
- if you select `erc20` you should see the screen to reserve namespace
- if you select `stake` you should see the flow for enabling stake (including namespace reservation)

## Deployment Plan

1. n/a

## Other Considerations

- n/a
